src/components/utilities.js

```diff
@@ -63,7 +63,8 @@ export function getAllValueField(fields) {
 export function forceSlashAfterHash(path) {
   let pathname = window.location.pathname;
   if (pathname[1] && pathname.includes('orcid_token')) {
-    window.location.href = window.location.origin + `#/${path}` + window.location.search;
+    pathname = pathname.replace('orcid_token', "")
+    window.location.href = window.location.origin + pathname + `#/${path}` + window.location.search;
```

Comment:
The original method ignores the rest part of the pathname. ex 'http://localhost:3000/ui/orcid_token" to "http://localhost:3000/#/orcid_token". The `ui` part is missed. The new method would include it.